entries/git-workflow-guidelines/commits.md

```diff
@@ -0,0 +1,213 @@
+
+But first, let's back up a little and look at the two phases of committing: When you want to commit some changes you have made, you first have to tell Git what changes you want to include in your commit (i.e., want to capture in the snapshot). This is done by adding them to what is known as the "staging area". Then, you save the snapshot by *committing* the staged changes with a message describing your changes.
+
+To understand these two phases, it might help to briefly go through how Git tracks changes (:warning: too much detail? :warning:). In the figure below (inspired by [this](https://learn.microsoft.com/en-us/azure/devops/repos/git/commits?view=azure-devops&tabs=visual-studio-2022#how-git-tracks-changes) post), it's visualised how Git tracks changes in three categories: **Unmodified**, **modified**, and **staged**.
+
+![How Git tracks file changes](../images/file-tracking-in-git.png)
+
+*Unmodified* files includes all tracked files that haven't been created or modified since the last commit. As soon as you save a new file or save an edited file, Git tracks that as being *modified*. When you are happy with your changes, you can *stage* the modifications by adding them to the *staging area*, which includes all changes that will be a part of the next commit. Adding changes to the staging area can, therefore, be said to be the first phase of a commit. In the second phase, you commit the staged changes, and Git saves a snapshot of your repository with these changes. This snapshot includes only the staged changes and the unmodified changes. Changes that haven't been staged are kept but not included in the snapshot.
+
+Note that you can add as many files and changes to the staging area as you want before you commit them. All changes in the staging area at the time of commit will be a part of the same commit. In this way, the staging area allows you to control which changes you want to include in a specific commit, so you can collect the changes connected to the same unit of work in one commit (we will return to what that means in the section [Atomic Commits](#atomic-commits) below).
+
+Commits are always created locally in the specific branch you are currently on. Therefore, it is important to check that you are on the right branch before you commit (check the [Branching](branching.md#branching-in-practice) post for how you check which branch you are currently on and how to change branch). After you have created your commits, they needs to be *pushed* to the remote repository (on GitHub or or any other version control service that uses Git) for anyone else to be able to see it.
+
```

Comment:
pushing and prs might not be relevant here? remove? or is it nice with a reminder that you have to push and get your changes reviewed?